packages/config/src/Config.js

```diff
@@ -65,10 +65,10 @@ export class Config {
     return this.#observables[key].subscribe(onChange);
   }
 
-  set({ value = null, key = null, onError = null }) {
+  set({ defaultValue = null, value = null, key = null, onError = null }) {
```

Comment:
what does defaultValue stands for?

Reply:
old value?

Reply:
It represents whatever we like, e.g:
https://github.com/logzio/gaia-hermes-ws/pull/8373
We passed our original config as default in order to merge our newValue to the originalValue on watchAll.